demos/noise_suppression_demo/python/noise_suppression_demo.py

```diff
@@ -117,24 +128,14 @@ def main():
         #add states to input
         for n in state_inp_names:
             if res:
-                inputs[n] = res[n.replace('inp', 'out')].buffer
+                inputs[n] = infer_request.get_tensor(n.replace('inp', 'out')).data
```

Comment:
```suggestion
                inputs[n] = copy.deepcopy(infer_request.get_tensor(n.replace('inp', 'out')).data)
```

Reply:
I have tried it, it doesn't work - the audio file will be "empty".

Reply:
very strange, but ok...